website/src/graphql/dailp/queries.graphql

```diff
@@ -415,11 +415,67 @@ mutation CurateWordAudio($input: CurateWordAudioInput!) {
 }
 
 mutation AddBookmark($documentId: UUID!) {
-  addBookmark(documentId: $documentId)
+  addBookmark(documentId: $documentId) {
```

Comment:
When you actually do your frontend PR, make sure to create a fragment on `AnnotatedDoc` with these fields. You should never have to copy+paste a list of fields to resolve. Let me know if you need help doing that.